plugins/community/timostamm-protobuf-ts/v2.8.2/buf.plugin.yaml

```diff
@@ -0,0 +1,17 @@
+version: v1
+name: buf.build/community/timostamm-protobuf-ts
+plugin_version: v2.8.2
+source_url: https://github.com/timostamm/protobuf-ts
+description: Generates Protobuf and RPC for TypeScript.
+output_languages:
+  - typescript
```

Comment:
It looks like it also has support for emitting JavaScript via options so we should add that here too.